package.json

```diff
@@ -10,8 +10,8 @@
   "description": "Tests for Azure REST API Specifications",
   "license": "MIT",
   "devDependencies": {
-    "@azure/avocado": "^0.2.0",
-    "@azure/oad": "^0.2.1",
+    "@azure/avocado": "^0.2.2",
```

Comment:
given the update is on the last digit for avocado, shouldn't our CI be picking up the latest published? why is this necessary?